packages/backend/src/App.ts

```diff
@@ -451,6 +451,8 @@ export default class App {
                 passportUser: { id: string; organization: string },
                 done,
             ) => {
+                // Set the organization tag so we can filter by it in Sentry
+                Sentry.setTag('organization', passportUser.organization);
```

Comment:
I'm thinking of setting it before each request handler. LMK if there's a better way of achieving this.